src/modules/client.py

```diff
@@ -39,24 +40,12 @@ def __init__(self, conf):
         self.is_open = True
 
     def _send_resources(self):
-        threads = []
-        thread_timeout = 0.05
-
         for res in ResourceMonitor.res_types:
             if self.is_open:
-                thread = Thread(
-                    target=self._send,
-                    args=(res,),
-                    name=res
-                )
-                threads.append(thread)
-                thread.start()
-                sleep(self.task_interval.total_seconds())
-
-        for thread in threads:
-            thread.join(thread_timeout)
-            if thread.is_alive():
-                logging.error("Timeout for getting " + str(thread.name))
+                try:
+                    self._send(res)
+                except TimeoutError:
+                    logging.error(res + " timeout")
```

Comment:
Pls use my existing log message (the one used for threading)

Reply:
I had done this too